PS-7991 Release Notes for 8.0.26-17 (8.0) (#4546)
percona/percona-server: doc/source/release-notes/Percona-Server-8.0.26-17.rst

* :mysqlbug:`104575`: In the PERFORMANCE_SCHEMA.Threads table, the ``srv_purge_thread`` and ``srv_worker_thread`` values are duplicated.
* :mysqlbug:`104387`: When using REGEX comparison, a CHARACTER_SET_MISMATCH error is thrown.
* :mysqlbug:`104576`: Accessing the second index in a partition table with many columns can create a high CPU load.

Find the full list of bug fixes and changes in `MySQL 8.0.26 Release Notes <https://dev.mysql.com/doc/relnotes/mysql/8.0/en/news-8-0-26.html>`__.

Deprecated Features
=================================================

The TokuDB Storage Engine was declared deprecated for Percona Server for MySQL. Starting with Percona Server 8.0.26-16, the plugins are available in the binary builds and packages but are disabled. The plugins will be removed from the binary builds and packages in a future version.

New options have been added to enable the plugins if they are needed to migrate the data to another storage engine.

The instructions on how to enable the plugins and more information are available at the beginning of each TokuDB topic.

Bugs Fixed
==================================================

> * To allow a single binary package, on platforms where ``libzbd`` is available or can be built (with 5.9 kernel and higher) ``libzbd`` statically linked into ha_rocksdb.so.

Reviewer: This reads kind of funny, maybe it's just me, but wouldn't some thing like this make a little more sense?
`To allow a single binary package, statically linked libzbd into ha_rocksdb.so. Requires 5.9 kernel and higher.`

Author: Rewritten.

* Fixed ZenFS issues with sysbench by using Debian 11 or the latest ``libzbd``.

* Fixed a sporadic [aborting on BG write error] assertion in rocksdb.bloomfilter3.

> * Fixed ``WITH_ZENFS_UTILITY`` CMake option is set to ``ON`` additional logic is added to the RocksDB *CMakeLists.txt* to ensure the ``libgflags`` library is installed on the system.

Reviewer: ```suggestion
* Fixed ``WITH_ZENFS_UTILITY`` CMake option when it is set to ``ON``.  Added logic to the RocksDB *CMakeLists.txt* to ensure the ``libgflags`` library is installed on the system.
```

Author: Updated.

> * Fixed when using the ``du`` system utility, the utility must be converted to computations based on the zenfs list output.

Reviewer: ```suggestion
* Fixed tests that rely on the ``du`` system utility. The ``du`` utility results must be converted to computations based on the zenfs list output.
```

Author: Updated.

> * Fixed by allowing zenfs mkfs to accept an existing pre-configured aux_path. 

Reviewer: ```suggestion
* Fixed zenfs mkfs command to allow it to accept a pre-existing aux_path. 
```

Author: Updated.
